### PR TITLE
Add jQuery to dependencies

### PR DESCRIPTION
**Pull request by Bajix**
See https://github.com/stealjs/steal/issues/971

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,5 +29,8 @@
     "jshint-stylish": "^1.0.0",
     "load-grunt-tasks": "^3.1.0",
     "time-grunt": "^1.0.0"
+  },
+  "dependencies": {
+    "jquery": "^3.1.1"
```
Comment (igorescobar) [inline package.json]: The correct version would be `^1.7.2` a not `3.x`. Could you fix it?

Comment (Bajix) [inline package.json]: That would make this semantically incompatible with newer versions. What's your supported version range?

Comment (igorescobar) [inline package.json]: @Bajix we support 1.7.2+ :)

**Comment by igorescobar** [comment]
Closed due inactivity.

**Pull request closed without merging**